python/doc/examples/probabilistic_modeling/distributions/plot_pointConditionalDistribution.py

```diff
@@ -0,0 +1,255 @@
+# In this example, we create some distributions conditionned by a particular value of its marginals.
+#
+# We consider the following cases:
+# - Case 1:  Bivariate distribution with continuous marginals,
+# - Case 2:  Bivariate distribution with continuous and discrete marginals,
+# - Case 3:  Trivariate distribution with continuous and discrete marginals,
+# - Case 4:  Extreme values conditioning.
+#
+# We illustrate the fact that the range of the conditioned distribution is updated.
+import openturns as ot
+import openturns.viewer as otv
+import openturns.experimental as otexp
+
+ot.ResourceMap.SetAsString("Contour-DefaultColorMapNorm", "rank")
+# sphinx_gallery_thumbnail_number =  5
+
+# %%
+# Case 1: Bivariate distribution with continuous marginals
+# --------------------------------------------------------
+#
+# Here, we consider the bivariate distribution of :math:`(X_0, X_1)` defined by:
+# .. math::
+#
+#   X_0 & \sim \Gamma(5, 0.5)
+#   X_1 & \sim \mbox{Student}(5)
+#   \mbox{copula} & \sim \mbox{ClaytonCopula}(2.5)
+#
+# We condition the marginal :math:`X_1` to be simultaneously equal to its quantiles of order
+# 0.05, 0.5 and 0.95. Then, we draw the resulting conditioned distribution of:
+# .. math::
+#
+#    X_0 | X_1 = x_1
+#
+# We define initial distribution of :math:`(X_0, X_1)`.
+coll_marg = [ot.Gamma(5.0, 0.5), ot.Student(5.0)]
+cop = ot.ClaytonCopula(2.5)
+dist_X = ot.JointDistribution(coll_marg, cop)
+
+# %%
+# We draw the PDF of :math:`(X_0, X_1)`.
+g_X = dist_X.drawPDF()
+g_X.setTitle(r"$(X_0, X_1)$: iso-lines PDF")
+g_X.setXTitle(r"$x_0$")
+g_X.setYTitle(r"$x_1$")
+
+# %%
+# We define the conditioned marginal, the list of the conditioning values and we create
+# all the conditioned distribution, whose PDF we draw on the same graph.
+#
+# We also print the updated range of the distribution of :math:`X_0`.
+cond_indices = [1]
+
+q_list = [0.05, 0.5, 0.95]
+cond_value_list = [coll_marg[1].computeQuantile(q)[0] for q in q_list]
+
+g_cond = coll_marg[0].drawPDF()
+g_cond.setLegends([r"dist of $X_0$"])
+for index, cond_value in enumerate(cond_value_list):
+    cond_value = cond_value_list[index]
+    cond_dist = otexp.PointConditionalDistribution(dist_X, cond_indices, [cond_value])
+    draw = cond_dist.drawPDF().getDrawable(0)
+    draw.setLegend(r"$x_1 = q($" + str(q_list[index]) + ")")
+    g_cond.add(draw)
+    print(
+        "Quantile of the cond value = ", q_list[index], "Range : ", cond_dist.getRange()
+    )
+
+g_cond.setTitle(r"PDF of $X_0|X_1 = x_1$")
+g_cond.setColors(ot.Drawable.BuildDefaultPalette(len(q_list) + 1))
+
+# %%
+# We gather both graphs into one grid.
+grid = ot.GridLayout(1, 2)
+grid.setGraph(0, 0, g_X)
+grid.setGraph(0, 1, g_cond)
+view = otv.View(grid)
+
+# %%
+# Case 2: Bivariate distribution with continuous and discrete marginals
+# ---------------------------------------------------------------------
+#
+# Here, we consider the bivariate distribution of :math:`(X_0, X_1)` defined by:
+# .. math::
```

Comment:
This is poorly rendered:

![image](https://github.com/user-attachments/assets/1687ba5f-80f5-4a69-b37f-400990d35823)